cotainr/pack.py

```diff
@@ -261,8 +262,10 @@ def _display_miniforge_license_for_acceptance(self, *, installer_path):
                 "\n",
             )
             logger.debug(f"The Miniforge displayed license is: {license_text}")
-            val = input(license_text)  # prompt user for acceptance of license terms
-            if val != "yes":
+            answer = util.answer_is_yes(
+                license_text
+            )  # prompt user for acceptance of license terms
+            if not answer:
```

Comment:
I still think
```suggestion
            # prompt user for acceptance of license terms
            if not util.answer_is_yes(license_text):
```
would look better while retaining the comment.

Reply:
Done.